jena-fuseki2/jena-fuseki-ui/tests/e2e/specs/datasets.cy.js

```diff
@@ -25,11 +25,17 @@ describe('datasets', () => {
   describe('without any datasets', () => {
     before(() => {
       // Special endpoint that clears the datasets data.
-      cy.request('/tests/reset')
+      cy.request({
+        url: '/tests/reset',
+        retryOnStatusCodeFailure: true
+      })
```

Comment:
It might be my environment, but for some unknown reason the fake Fuseki used in the e2e tests started failing intermittently.

Sometimes everything worked fine, sometimes the second or third test failed.

Tried debugging it, changing settings, but couldn't understand what's happening. Looked at the issues of Json-server but didn't find anything helpful (the test fails with HTTP error 500 on the Json-server reply).

Cypress contains a setting to retry (4 times by default), which fixed the test on my environemnt. Not sure what could have caused it.